wrappers/internal/service_status.go

```diff
@@ -45,6 +51,14 @@ func (s *ServiceStatus) ServiceUnitStatus() *systemd.UnitStatus {
 	return s.service
 }
 
+func (s *ServiceStatus) ActivatorUnitStatuses() []*systemd.UnitStatus {
+	return s.activators
+}
+
+func (s *ServiceStatus) User() bool {
```

Comment:
these should have a doc comment

Reply:
doc comments are added for this interface